superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -614,214 +579,145 @@ class SqlEditor extends React.PureComponent {
-      this.state.createAs === CtasEnum.VIEW
-        ? 'CREATE VIEW AS'
-        : 'CREATE TABLE AS';
-
-    const createModalPlaceHolder =
-      this.state.createAs === CtasEnum.VIEW
-        ? 'Specify name to CREATE VIEW AS schema in: public'
-        : 'Specify name to CREATE TABLE AS schema in: public';
-    const leftBarStateClass = this.props.hideLeftBar
-      ? 'schemaPane-exit-done'
-      : 'schemaPane-enter-done';
-    return (
-      <div ref={this.sqlEditorRef} className="SqlEditor">
-        <CSSTransition
-          classNames="schemaPane"
-          in={!this.props.hideLeftBar}
-          timeout={300}
+  const createViewModalTitle =
+    createAs === CtasEnum.VIEW ? 'CREATE VIEW AS' : 'CREATE TABLE AS';
+
+  const createModalPlaceHolder =
+    createAs === CtasEnum.VIEW
+      ? t('Specify name to CREATE VIEW AS schema in: public')
+      : t('Specify name to CREATE TABLE AS schema in: public');
+
+  const leftBarStateClass = hideLeftBar
+    ? 'schemaPane-exit-done'
+    : 'schemaPane-enter-done';
+  return (
+    <div ref={sqlEditorRef} className="SqlEditor">
+      <CSSTransition classNames="schemaPane" in={!hideLeftBar} timeout={300}>
+        <ResizableSidebar
+          id={`sqllab:${queryEditor.id}`}
+          minWidth={SQL_EDITOR_LEFTBAR_WIDTH}
+          initialWidth={SQL_EDITOR_LEFTBAR_WIDTH}
+          enable={!hideLeftBar}
         >
-          <ResizableSidebar
-            id={`sqllab:${this.props.queryEditor.id}`}
-            minWidth={SQL_EDITOR_LEFTBAR_WIDTH}
-            initialWidth={SQL_EDITOR_LEFTBAR_WIDTH}
-            enable={!this.props.hideLeftBar}
-          >
-            {adjustedWidth => (
-              <StyledSidebar
-                className={`schemaPane ${leftBarStateClass}`}
-                width={adjustedWidth}
-                hide={this.props.hideLeftBar}
+          {adjustedWidth => (
+            <StyledSidebar
+              className={`schemaPane ${leftBarStateClass}`}
+              width={adjustedWidth}
+              hide={hideLeftBar}
+            >
+              <SqlEditorLeftBar
+                database={database}
+                queryEditor={queryEditor}
+                tables={tables}
+                actions={actions}
+                setEmptyState={bool => setShowEmptyState(bool)}
+              />
+            </StyledSidebar>
+          )}
+        </ResizableSidebar>
+      </CSSTransition>
+      {showEmptyState ? (
+        <EmptyStateBig
+          image="vector.svg"
+          title={t('Select a database to write a query')}
+          description={t(
+            'Choose one of the available databases from the panel on the left.',
+          )}
+        />
+      ) : (
+        queryPane()
+      )}
+      <Modal
+        visible={showCreateAsModal}
+        title={t(createViewModalTitle)}
+        onHide={() => setShowCreateAsModal(false)}
+        footer={
+          <>
+            <Button onClick={() => setShowCreateAsModal(false)}>Cancel</Button>
```

Comment:
```suggestion
            <Button onClick={() => setShowCreateAsModal(false)}>{t('Cancel')}</Button>
```
This string needs to be translated